client/src/components/Registration/Registration.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+import { Form, Button, Row, Col } from "react-bootstrap";
+const axios = require("axios");
+
+function Resisration() {
```

Comment:
Looks nice Miku! One thing is that it would be good to spell-check some words here like 
`function Resisration` should be spelled `Registration`. I am only asking so we don't get bugs in the future  👍

Reply:
Thank you Tarnner ! I fixed spelling.